### DISCUSSION TITLE
Is anyone using useGracefulShutdown in a production enviornment?

### DISCUSSION THREAD

**Discussion by mpaul31** [Azure/azure-functions-durable-extension#1816] [Q&A]
Knowing this is a _preview_ settings, just wanted to get a quick pulse and see if this is being used out in the wild and if so, have your metrics been favorable?

**Comment by cgillum** [comment]
I didn't realize this was still in preview. @ConnorMcMahon would it make sense to remove this from preview, given we've already gotten feedback that it's working well for customers that needed it?

**Reply by mpaul31** [reply]
Didn't see much in the documentation, but is this just an opt-in setting? Or is the use of cancellation tokens involved?
